CHANGES.rst

```diff
@@ -1,6 +1,18 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-20)
+--------------------------
+
+- Adds user secrets management mechanism.
+- Adds `--workflow` flag for specifying workflow for `run` and `create`
+  commands.
+- Adds version command.
+- Modifies file upload to use `requests` instead of Bravado.
+- Upgrades cwltool to 1.0.20191022103248.
```

Comment:
To keep consistency, `cwltool` here should be between backquotes (as `requests` above).

Reply:
Yes, I'll amend the backquotes while updating the wording a little... Aiming for the release tomorrow.